GolfExample_TCAPI/scripts/common.js

```diff
@@ -26,5 +52,10 @@ GolfExample.getContext = function(parentActivityId) {
             id: parentActivityId
         };
     }
+    if (isAssessment){
+        ctx.contextActivities.grouping.push({
```

Comment:
I believe that isAssessment applies to the questions and to the assessment itself. 

Certainly the resulting statements have the assessment as a grouping for both the assessment and the questions; the questions have the assessment as the parent. The code seems to be doing what it's intended to do. 

The next question is "why is the assessment used as the grouping as well as the parent?" The answer seems to be this slack conversation with Ben on 27th Nov (Thanksgiving): https://rs-oem.slack.com/archives/canformance/p1417100716000002 (apologies this isn't accessible to people outside Rustici Software).